bodo/pandas/_physical_conv.cpp

```diff
@@ -53,9 +57,116 @@ void PhysicalPlanBuilder::Visit(duckdb::LogicalProjection& op) {
     this->active_pipeline->AddOperator(physical_op);
 }
 
+// Generic dynamic_cast for std::unique_ptr
+template <typename Derived, typename Base>
+duckdb::unique_ptr<Derived> dynamic_cast_unique_ptr(
+    duckdb::unique_ptr<Base>& base_ptr) {
```

Comment:
```suggestion
    duckdb::unique_ptr<Base>&& base_ptr) {
```
Since we're taking ownership

Reply:
Done.